src/main/java/org/xstefank/verification/Verification.java

```diff
@@ -0,0 +1,9 @@
+package org.xstefank.verification;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public interface Verification {
+
+    boolean verify(FormatConfig formatConfig);
+    String getErrorMessage();
```

Comment:
it would be sufficient to document here how `getErrorMessage` behaves in case `verify` wasn't called or the verification was successful, maybe it would be better to ask if it does make sense as a separate method